### PR TITLE
feat: catch non existant validator n action error

### LINKED ISSUE #224: No error feedback when validate.do has an invalid key
If a `validate.do` key includes an invalid value, Mergeable silently does not do anything. It does not report red, yellow, or green at all.

If I set invalid version, Mergeable nicely reports “Invalid configuration”.

If I use `mussst_exclude` instead of `must_exclude`, Mergeable reports this is not supported.

So this is specific to `validate.do` key.

Minimally similar to #154.

This config triggers Mergeable correctly:

```yml
version: 2
mergeable:
  - when: pull_request.opened, pull_request.edited, pull_request.synchronize
    validate:
      - do: description
        must_exclude:
          regex: 'DO NOT MERGE|WIP'
```

Here I changed `description` to `thing` by mistake and Mergeable is silent:

```yml
version: 2
mergeable:
  - when: pull_request.opened, pull_request.edited, pull_request.synchronize
    validate:
      - do: thing
        must_exclude:
          regex: 'DO NOT MERGE|WIP'
```

### PR DESCRIPTION
**Pull request by shine2lay**
closes #224 
output : 
image

@jusx please pay a closer attention to this PR, it has some refactoring involved for better separation

### CODE REVIEW

File: lib/registry.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      throw new Error('Actions have thrown ' + err)
+    }
+  })
+}
+
+module.exports = {
+  registerValidators,
+  registerActions,
+  registerValidatorsAndActions
+}
```
Comment (jusx) [inline lib/registry.js]: In general I strongly suggest keeping the order of the parameters consistent. You have register as the first or second parameter interchangeably.

File: lib/flex.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const Configuration = require('./configuration/configuration')
 const extractValidationStats = require('./stats/extractValidationStats')
 const Checks = require('./actions/checks')
+const { registerValidatorsAndActions } = require('./registry')
```
Comment (jusx) [inline lib/flex.js]: I personally would much prefer doing a require without the scoping. i.e. 

`const registry = require('.registry')`

it is much more obvious that `registry.registerValidatorsAndActions` is part of another module/object then reading `registerValidatorsAndActions` that can be either a function within this same file or coming from somewhere else.

minor comment and personal preference.

File: lib/registry.js (hunk 1 above)
Comment (shine2lay) [reply lib/registry.js]: make sense, good catch

File: lib/flex.js (hunk 2 above)
Comment (shine2lay) [reply lib/flex.js]: done.

File: lib/registry.js (hunk 1 above)
Comment (shine2lay) [reply lib/registry.js]: done

**Pull request merged**